### DISCUSSION TITLE
Comunicazioni verso altri enti

### DISCUSSION THREAD

**Discussion by Hdemografici** [italia/ansc#968] [Supporto servizi cooperativi] [answered]
Buonasera,
in riferimento alle comunicazioni da inviare ad altri enti a seguito redazione di un atto (es. comunicazione per l'Azienda Sanitaria a seguito "Dichiarazione nei termini di filiazione fuori dal matrimonio resa all'USC dal padre con presenza madre" - caso uso 11213000) 
si chiede se un Comune, che ha adottato ANSC con i servizi cooperativi, debba necessariamente eseguire tali adempimenti utilizzando i testi messi a disposizione dal sistema ANSC (utilizzando il servizio R003) oppure possa scegliere di continuare a utilizzare il corrispettivo testo messo a disposizione dalla software house.

Si sfrutta occasione per aggiungere a riguardo:
esiste una mappatura indicante l'associazione tra caso d'uso e relative comunicazioni?

grazie

**Comment by asqueglia** [comment, marked as answer]
Buongiorno in merito alle comunicazioni destinate ad enti esterni, comunichiamo che il sistema mette a disposizione solo alcune comunicazioni di ausilio, di carattere generale,; potete per il momento continuare ad utilizzare le comunicazioni rivolte ad enti terzi (non comuni) predisposte dal vostro gestionale in quanto sicuramente più esaustive.
